CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Fixed
+- `ci.yaml`: Remove outdated comment about mirrord schema.
```

Comment:
In my opinion it's better not to have a changelog entry for this minor change.
I wouldn't want users that are interested what's new in the next mirrord release to read about a deleted comment in  our CI script 😁